rustls/src/error.rs

```diff
@@ -102,6 +102,12 @@ pub enum Error {
     /// [`keys_match`]: crate::crypto::signer::CertifiedKey::keys_match
     InconsistentKeys(InconsistentKeys),
 
+    /// A asynchronous operation is pending.
+    ///
+    /// Once the relevant operation completes, re-try this call to
+    /// advance the handshake.
```

Comment:
nit: I feel like "this call" might be ambiguous. Perhaps "re-try the operation that produced the `PendingOperation` error?" too verbose?